README.md

```diff
@@ -1,2 +1,69 @@
 # invidious-stripped-down
-Invidious api just for api dash manifest and latest_version
+
+Invidious api just for api dash manifest and `latest_version`.
+
+## Variables
+
+Environment variables are used to configure the application.
+
+| Variable        | Default     | Description                                              |
+| --------------- | ----------- | -------------------------------------------------------- |
+| `DNS_ORDER`     | `verbatim`  | DNS order for lookups. Other possible value: `ipv4first` |
+| `HOST_PROXY`    | `undefined` | Host proxy                                               |
+| `KEYV_MAX_SIZE` | `5000`      | Keyv max size                                            |
+| `KEYV_ADDRESS`  | `undefined` | Keyv address, In-memory if not defined                   |
+| `BIND_PORT`     | `3000`      | Port                                                     |
+| `BIND_ADDRESS`  | `0.0.0.0`   | Address                                                  |
+| `HMAC_KEY`      | `undefined` | HMAC key for invidious                                   |
+
+## Usage
+
+Set environment variables as needed.
+
+### Docker
+
+```bash
+docker run -d \
+  -e HOST_PROXY=invidious.io \
+  -e HMAC_KEY=00000000000000000000 \
+  -p 3000:3000 \
+  --name invidious-stripped-down \
+  quay.io/skyluker4/invidious-stripped-down
```

Comment:
maybe not advertise your docker image. you can say use these commands for building the local docker image.